docs/openshift/important-changes-by-release.md

```diff
@@ -0,0 +1,7 @@
+# Important Changes by OCP Release
+
+## 4.2
+
+### New Global Catalog Namespace
+
+The default _GCN_ (global catalog namespace) has been changed from OLM's namespace, `openshift-operator-lifecycle-manager`, to `operator-marketplace`. As a quick refresher, the GCN is a special namespace that contains operator catalogs that can be subscribed to from any namespace in the cluster. The important positive implication of this change is that default catalogs in the `operator-marketplace` namespace (`redhat-operators`, `certified-operators`, etc...) can be subscribed to from all namespaces without further configuration. The negative implication is that any subscriptions to catalogs in the `openshift-operator-lifecycle-manager` namespace must be co-located in that namespace.
```

Comment:
I feel like there should be: 

1.  A doc with a section on the global catalog namespace
2. This Changelog which references that section (i.e. the "refresher" should refresh something covered more in-depth elsewhere

Also I think we should avoid introducing more acronymns like GCN 😄

Reply:
> I feel like there...

These were my thoughts exactly. My plan was to add immediately useful info about the question we've been asked first, then come around and add the global catalog section in a follow up. Is that okay?

I can remove the acronym, I just didn't want to type it over and over again here.